include/rocksdb/env.h

```diff
@@ -738,6 +739,9 @@ class SequentialFile {
         "SequentialFile::PositionedRead() not supported.");
   }
 
+  virtual Status GetFileSize(uint64_t& /* size */) const {
+    return Status::NotSupported("GetFileSize");
```

Comment:
nit: I'd stay aligned with how existing `Status::NotSupported` msgs above are - `return Status::NotSupported("SequentialFile::GetFileSize not supported.");`

Reply:
Ah okay, copy paste mistake :)